lib/models/models.dart

```diff
@@ -136,6 +138,45 @@ class BalanceModel {
   int get hashCode => super.hashCode;
 }
 
+class RateModel {
+  final double numericQuantity;
```

Comment:
Quantity seems to be a misleading name. The model is Rate, but the contents of it is Quantity? What is stored here?